config/application.rb

```diff
@@ -70,7 +70,7 @@ class Application < Rails::Application
     config.middleware.use Rack::Cors do
       allow do
         origins '*'
-        resource '/api/*', headers: :any, methods: [:get, :post, :options, :put, :delete]
+        resource '/api/*', headers: :any, methods: [:get, :post, :options, :put, :delete], expose: ["Link"]
```

Comment:
Line is too long. [107/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.